vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutIT.java

```diff
@@ -69,27 +67,6 @@ public void dialogOpenedWithListenOnShortcut_sameShortcutListeningOnUi_focusDeci
     }
 
     @Test
-    @Ignore("flaky test see https://github.com/vaadin/flow-components/issues/777")
-    public void dialogOpenedWithShortcutNoListenOn_sameShortcutListeningOnUi_bothExecuted() {
```

Comment:
Kind of hard to decipher what these tests are for. This one tested that a shortcut is both executed on the UI and on the dialog when a modal dialog is opened. That doesn't work anymore with server-side modality, so I assume this is obsolete.